docs/document/website/versioned_docs/version-1.8.0/guides/attachment-customization.md

```diff
@@ -0,0 +1,175 @@
+---
```

Comment:
Please update in the same way with the docs/document/docs/guides/attachment-customization.md.